### ISSUE TITLE
Setting background on VGSEditText programatically doesn't remove the underbar

### ISSUE THREAD

**Issue by AungThiha** [verygoodsecurity/vgs-collect-android#197]
**Describe the bug**
Setting background on VGSEditText programatically doesn't remove the underbar.
It does if I statically set the background using `android:background` but I need to update the background programmatically based on the state of the input.

**To Reproduce**

Activity.kt
```
vgsEditText.background = AppCompatResources.getDrawable(context, R.drawable.bg_card_pin)
```

bg_card_pin.xml
```
<?xml version="1.0" encoding="utf-8"?>
<shape xmlns:android="http://schemas.android.com/apk/res/android">
    <solid android:color="#623568" />

    <corners
        android:radius="8dp" />
</shape>
```

layout.xml
```
<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    android:layout_width="match_parent"
    android:layout_height="match_parent"
    android:background="#6C3F72"
    android:orientation="vertical"
    android:padding="16dp">
    <com.verygoodsecurity.vgscollect.widget.VGSEditText
        android:id="@+id/vgsEditText"
        android:layout_width="match_parent"
        android:layout_height="wrap_content"
        app:enableValidation="true"
        app:fieldName="pin" />
</LinearLayout>
```

![204769641_938856926965193_1278576237545771333_n](https://user-images.githubusercontent.com/4388183/123043268-3d753480-d422-11eb-8855-d4d2ba00092c.jpeg)

**Expected behavior**
It shouldn't display underbar

**Environment (please complete the following information):**
 - SDK Version: 1.6.9
 - Android Version: 11
 - Device: Samsung Galaxy S20

**Comment by AungThiha** [comment]
another bug related to this is if the background is statically set using `android:background`, it can't be changed programmatically anymore. Do you need need to open another issue for this?

**Comment by Endorf** [comment]
Thank you for mentioning us about this issues. We will take a look on both.

**Comment by Endorf** [comment]
We released a new version -  `v1.6.10` with the fix to this issue.

**Comment by AungThiha** [comment]
Thanks, @Endorf. I just tested it. It works
